client/jetpack-connect/style.scss

```diff
@@ -686,3 +686,29 @@
 		padding-right: 319px;
 	}
 }
+
+.jetpack-connect__plan-info {
+	margin-top: 35px;
+	text-align: center;
+	width: 93%;
```

Comment:
The `93%` width seems a bit off to me. It won't be consistent through various mobile devices. You will see the problem occurs when you start resizing down your screen, and comparing the width of the card with the width that the plans take - they'll be inconsistent:

![](https://cldup.com/VdI_OWKHZy.png)

I'd refrain from that by using the default width of `100%` and just apply some margins. There would have to be different margins for the different breakpoints, because the plans above will behave differently.

This comment applies to all the occurrences where we specify `width` to `.jetpack-connect__plan-info`

Reply:
As spoken off-GH, atm this cannot be solved elegantly. We'll reshape the design to support varying viewports in style :)